src/pickup.cpp

```diff
@@ -625,25 +641,26 @@ void Pickup::pick_up( const tripoint &pos, int min )
         // Now print the two lists; those on the ground and about to be added to inv
         // Continue until we hit return or space
         do {
-            static const std::string pickup_chars =
-                "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ:;";
+            static const std::string pickup_chars = ctxt.get_available_single_char_hotkeys(
+                "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ:;");
```

Comment:
Yea, nifty little benefit of using input_context for hotkeys. I have a WIP PR to fix this in the main inventory from a year or two ago, but never got around to finishing it. The main inventory screen turned out to be a rabbit hole that had no bottom. :)

But don't make it static though, like it is here. I'll remove static in the next diff.